src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -637,6 +637,16 @@ public static XmlSerializer[] FromTypes(Type[] types)
         {
             if (types == null)
                 return Array.Empty<XmlSerializer>();
+
+#if NET_NATIVE
+            var serializers = new List<XmlSerializer>();
```

Comment:
Why couldn't this be something like...

```c#
var serializers = new XmlSerializer[types.Length];
for (int i = 0; i < types.Length; i++)
{
    serializers[i] = new XmlSerializer(types[i]);
}

return serializers;
```

... to avoid the unnecessary `List<T>` allocation and internal array allocations inside `List<T>` as items are added and it needs to grow, plus another final `ToArray()` array allocation?